resources/classdetail/image/compute_image.go

```diff
@@ -0,0 +1,85 @@
+package classdetail
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"strings"
+)
+
+type computeImage struct {
+	CreationTimestamp string
+	Image             string
+	Family            string
+	DiskSizeGib       int64
+}
+
+type imageByDate []computeImage
+
+var imagesByFamily map[string][]computeImage
```

Comment:
Nit: add a comment for the package variables, when are these initialized, concurrency (non-)concerns, etc.